src/service/packet_verifier.proto

```diff
@@ -34,4 +34,5 @@ message valid_data_transfer_session {
   // session in
   uint64 last_timestamp = 7;
   uint64 rewardable_bytes = 8;
+  uint64 burn_timestamp = 9;
```

Comment:
does this field store a timestamp in seconds or milliseconds?

Reply:
Good question! 2x checking

Reply:
`Timestamp in millis dated when burn transaction is confirmed`